src/components/form/FormFieldDonation.vue

```diff
@@ -34,10 +37,19 @@ export default defineComponent({
   },
   emits: ['update:donation'],
   setup(props, { emit }) {
-    const defaultPaymentAmountMin = parseInt(
-      rideToWorkByBikeConfig.entryFeePaymentMin,
-    );
-    const amount = ref<number>(defaultPaymentAmountMin);
+    const challengeStore = useChallengeStore();
+    const defaultPaymentAmountMin = computed(() => {
+      const currentPriceLevels = challengeStore.getCurrentPriceLevels;
```

Comment:
Same as my comment https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/814/files#r1899911173.

Reply:
Fixed with commit https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/814/commits/7671dbde12c708b595cf8e9ceff791533fa5bd0e.